src/app/server/OnboardingCodesUtil.cpp

```diff
@@ -28,8 +28,9 @@
 #include <support/ScopedBuffer.h>
 #include <support/logging/CHIPLogging.h>
 
-constexpr char qrCodeBaseUrl[]                   = "https://dhrishi.github.io/connectedhomeip/qrcode.html";
-constexpr char specialCharsUnreservedInRfc3986[] = "-._~";
+constexpr char kQrCodeBaseUrl[]                   = "https://dhrishi.github.io/connectedhomeip/qrcode.html";
+constexpr char kUrlDataAssignmentPhrase[]         = "?data=";
```

Comment:
nit: this could in fact be merged with `kQrCodeBaseUrl` to simplify the code a little bit.

Reply:
you can actually ignore this comment, it's super nit :)

Reply:
I did it intentionally, as community wants to soon change base URL and I left it exposed for them.